.env.default

```diff
@@ -13,3 +13,6 @@ export TEST_DATABASE_HOST=
 export TEST_DATABASE_NAME=
 export TEST_DATABASE_USER=
 export TEST_DATABASE_USER_PASSWORD=
+
+
+export BRAINSTORMING_TEASERS=true
```

Comment:
Just a thought: What about inverting the logic of this env var `MW_FEATURE_SKIP_BRAINSTORMING_TEASERS`. If this setting set with any value then this means that it is enabled. If it is`nil` or empty, than it should not be considered set.

Reply:
that was my idea at first. But I dont like negative logic which often leads to confusion. So either a feature exists or not.